_posts/2020-10-20-SUGCH-2020-3.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: SUGCH 2020/3 - Sitecore Messaging Framework
+---
+
+Our third meetup of 2020 was all about messaging. We were happy to have 22 Sitecore enthusiasts attending this online meeting and listening to our guest speaker's interesting presentation.
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Less than an hour left until the next <a href="https://twitter.com/hashtag/SUGCH?src=hash&amp;ref_src=twsrc%5Etfw">#SUGCH</a> meetup. Join us and <a href="https://twitter.com/mitya_1988?ref_src=twsrc%5Etfw">@mitya_1988</a> who will talk about <a href="https://twitter.com/hashtag/Sitecore?src=hash&amp;ref_src=twsrc%5Etfw">#Sitecore</a>&#39;s Messaging Framework. It&#39;s not too late, register here <a href="https://t.co/35y1cjWoht">https://t.co/35y1cjWoht</a></p>&mdash; Tobias Studer (@studert) <a href="https://twitter.com/studert/status/1331241004670783489?ref_src=twsrc%5Etfw">November 24, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Presentations
+
+### Sitecore Messaging Framework
+
+Our guest *Mihály Árvai* showed us a hidden treasure in the form of the *Sitecore Messaging Framework*. After an introduction on messaging basics, we quickly dived into some personalized code:
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">We love code examples where we are mentioned by name! Thank you and hello back <a href="https://twitter.com/mitya_1988?ref_src=twsrc%5Etfw">@mitya_1988</a> &lt;3 <a href="https://twitter.com/hashtag/SUGCH?src=hash&amp;ref_src=twsrc%5Etfw">#SUGCH</a> <a href="https://twitter.com/hashtag/Sitecore?src=hash&amp;ref_src=twsrc%5Etfw">#Sitecore</a> <a href="https://t.co/1rqWC6yfvT">pic.twitter.com/1rqWC6yfvT</a></p>&mdash; Tobias Studer (@studert) <a href="https://twitter.com/studert/status/1331259073296216064?ref_src=twsrc%5Etfw">November 24, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Mihály's demo included communication from a CM to CD instance and vice-versa using Sitecore's Messaging Framework and an Azure Service Bus. Thank you for sharing these hidden treasures with us!
+
+Mihály can be found on:
```

Comment:
I'd probably replace L20-_28 with:
[Code examples supplementing the presentation](https://github.com/Mitya88/SitecoreMessagingSamples) can be found on Github.

-> Twitter handle linked, github repo linked, medium advertised during presentation already.